front/src/modules/simulationResult/components/SpeedSpaceChart/useSpeedSpaceChart.ts

```diff
@@ -7,21 +7,20 @@ import type { PathPropertiesFormatted } from 'applications/operationalStudies/ty
 import { preparePathPropertiesData } from 'applications/operationalStudies/utils';
 import {
   osrdEditoastApi,
-  type PacedTrainResponse,
   type PathfindingResultSuccess,
   type SimulationResponse,
-  type TrainScheduleResult,
 } from 'common/api/osrdEditoastApi';
 import { useInfraID } from 'common/osrdContext';
 import usePathProperties from 'modules/pathfinding/hooks/usePathProperties';
 import formatPowerRestrictionRangesWithHandled from 'modules/powerRestriction/helpers/formatPowerRestrictionRangesWithHandled';
 import type { SpeedSpaceChartData } from 'modules/simulationResult/types';
+import type { TimetableItemWithTimetableId } from 'reducers/osrdconf/types';
 
 import { updateChartSynchronizerTrainData } from '../ChartSynchronizer/utils';
 
 /** Prepare data needed for speedSpaceChart */
 const useSpeedSpaceChart = (
-  timetableItemResponse?: TrainScheduleResult | PacedTrainResponse,
+  timetableItemWithTimetableId?: TimetableItemWithTimetableId,
```

Comment:
timetableItem is ok for a name I think :) 
```suggestion
  timetableItem?: TimetableItemWithTimetableId,
```

Reply:
Sure